sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/SuppressReviveManagerTest.java

```diff
@@ -141,6 +149,21 @@ public Boolean call() throws Exception {
                 });
     }
 
+    private static void waitStateNeverOccurs(
+            SuppressReviveManager suppressReviveManager,
+            SuppressReviveManager.State state) {
+
+        Awaitility.await()
```

Comment:
if SuppressReviveManager entered WAITING_FOR_OFFER, i assume it wouldn't automatically go back to REVIVED, would it? if so then this 5s delay could be omitted...

quick way to check would be reverting the SuppressReviveManager change, and disabling this wait, and then seeing if the assert on L117 is enough to catch the issue